File: FirstBit Workspace/C Programming/Assignments/Assignment 8/Q2.c
//Search the given number in array.
#include<stdio.h>
void main(){
	
	int arr[10], num, flag=0, flag1=0;
	
	//storing numbers in a array
	printf("Enter 10 numbers to store in a array: ");
	for(int i=0;i<10;i++)
		scanf("%d",&arr[i]);
		
	printf("\nYour array: ");
	for(int i=0;i<10;i++)
		printf("%d ",arr[i]);
		
	printf("\nEnter number to search in a array: ");
	scanf("%d",&num);
	for(int i=0;i<10;i++){
		if(num==arr[i]){
			flag=1;
		}
		if(flag==1){
			printf("\nNumber found at index %d: %d",i,arr[i]);
			flag1=1;
		}
		flag=0;
	}
	
	if(flag1==0)
		printf("\nNumber not present in a array");

}

File: FirstBit Workspace/Core Java/In Class Work/ArrayCProgramsInJava/src/Q2.java
import java.util.Scanner;

public class Q2 {

	public static void main(String[] args) {
		
		//Search the given number in array.
		
				//array reference 
				int[] array;
				
				//initialization
				array = new int[5];
				
				Scanner sc = new Scanner(System.in);
				
				System.out.println("Enter elements to store in array: ");
				
				//store elements
				for(int i=0; i<array.length; i++) {
					array[i] = sc.nextInt(); 
				}
				
				System.out.println("\nElements stored in array: ");
				//display array
				for(int ele:array) {
					System.out.println(ele);
				}
				
				System.out.println("\nEnter Number to search in array: ");
				int numToSearch = sc.nextInt();
				
				boolean flag = true;
				
				for(int i=0; i<array.length; i++) {
					if(numToSearch == array[i]) {
						System.out.println("\nNumber found at index " + i);
						flag = false;
					}
				}
				
				if(flag) System.out.println("\nNumber not found in the array.");
				
				sc.close();

	}

}
